cmake/kokkos_enable_options.cmake

```diff
@@ -55,6 +55,7 @@ KOKKOS_ENABLE_OPTION(COMPILER_WARNINGS    OFF "Whether to print all compiler war
 KOKKOS_ENABLE_OPTION(PROFILING_LOAD_PRINT OFF "Whether to print information about which profiling tools got loaded")
 KOKKOS_ENABLE_OPTION(TUNING               OFF "Whether to create bindings for tuning tools")
 KOKKOS_ENABLE_OPTION(AGGRESSIVE_VECTORIZATION OFF "Whether to aggressively vectorize loops")
+KOKKOS_ENABLE_OPTION(NATIVE_OPENMP        ON  "Whether to enable refactored OpenMP backend implementation")
```

Comment:
Enabled temporarily to get the CI feedback.
This should be either `OFF` by default or (preferably) removed completely.